main/webapp/modules/core/styles/dialogs/clustering-dialog.css

```
@@ -166,3 +166,78 @@ table.clustering-dialog-entry-table a:hover {
   text-decoration: none;
   visibility: hidden;
 }
+
+.binning-controls {
+  position: relative;
+  display: inline-block;
+}
+
+.keyingDropdown {
+  display: inline-block;
+}
+
+#keyingDropdownBtn {
+  min-width: 130px;
+  text-align: left;
+  background-color: #ffffff;
+  border: 1px solid #727272;
+}
+
+.keyingFunctionSelector {
```

Nitpick: for consistency with the rest of the class names, you could stick to this syntax:
```suggestion
.keying-function-selector {
```
and similarly for other CSS rules in this file